components/compass-runtime-agent/internal/compassconnection/connector.go

```diff
@@ -26,7 +26,7 @@ const (
 //go:generate mockery --name=Connector
 type Connector interface {
 	EstablishConnection(connectorURL, token string) (EstablishedConnection, error)
-	MaintainConnection(renewCert bool) (*certificates.Credentials, v1alpha1.ManagementInfo, error)
+	MaintainConnection(renewCert bool, credentialsExist bool) (*certificates.Credentials, v1alpha1.ManagementInfo, error)
```

Comment:
I think we should also regenerate the mock of the `Connector` interface

Reply:
Done